### ISSUE TITLE
bbb-exporter stops with scalelite when no meetings

### ISSUE THREAD

**Issue by merterhk** [greenstatic/bigbluebutton-exporter#55]
Because scalelite dont return `<meetings/>` tag when there are no meeting. If there is one or more meeting it runs correctly.

Docker log:
```
2020-09-11 10:42:19,901 [DEBUG]: https://***.edu.tr:443 "GET /bigbluebutton/api/getMeetings?checksum=*** HTTP/1.1" 200 None
----------------------------------------

Exception happened during processing of request from ('172.18.0.3', 35978)
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/socketserver.py", line 650, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/local/lib/python3.8/socketserver.py", line 360, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/local/lib/python3.8/socketserver.py", line 720, in __init__
    self.handle()
  File "/usr/local/lib/python3.8/http/server.py", line 427, in handle
    self.handle_one_request()
  File "/usr/local/lib/python3.8/http/server.py", line 415, in handle_one_request
    method()
  File "/usr/local/lib/python3.8/site-packages/prometheus_client/exposition.py", line 152, in do_GET
    output = encoder(registry)
  File "/usr/local/lib/python3.8/site-packages/prometheus_client/openmetrics/exposition.py", line 14, in generate_latest
    for metric in registry.collect():
  File "/usr/local/lib/python3.8/site-packages/prometheus_client/registry.py", line 75, in collect
    for metric in collector.collect():
  File "/app/collector.py", line 55, in collect
    meetings, meetings_data_latency = execution_duration(api.get_meetings)()
  File "/app/helpers.py", line 19, in wrapper
    result = fun(*args, **kwargs)
  File "/app/api.py", line 14, in get_meetings
    if data['response']['meetings'] is None:
KeyError: 'meetings'
```

Scalelite API getMeetings function response:
```
<?xml version="1.0"?>
<response>
  <returncode>SUCCESS</returncode>
  <messageKey>noMeetings</messageKey>
  <message>No meetings were found on this server.</message>
</response>
```

Other BBB API getMeetings function response:
```
<response>
<returncode>SUCCESS</returncode>
<meetings/>
<messageKey>noMeetings</messageKey>
<message>no meetings were found on this server</message>
</response>
```

**Comment by merterhk** [comment]
Adding these 2 lines to api.py line 13 solved problem.

```
    if 'meetings' not in data['response']:
        return []

```

**Comment by greenstatic** [comment]
Could you maybe create a PR for this? :)

**Comment by faust64** [comment]
Here you go, @greenstatic

**Comment by greenstatic** [comment]
Thanks @merterhk for the bug report and fix. And thank you @faust64 for the PR :)

Can one of you confirm that the [current master](https://github.com/greenstatic/bigbluebutton-exporter/tree/7533bb04881b3aa8df782863fc85b3f78957b1ae) build solves your bug?

You can replace the docker image tag to master, pull the new image and recreate the docker container.

Note, the version tag has not changed yet, don't let that fool you.

**Comment by merterhk** [comment]
Updating docker image with greenstatic/bigbluebutton-exporter:latest solved problem. Thanks @greenstatic @faust64.

**Comment by greenstatic** [comment]
This is officially fixed in release [v0.5.2](https://github.com/greenstatic/bigbluebutton-exporter/releases/tag/v0.5.2). Please change your docker image tags to `v0.5.2` and create your container.
